_includes/specials/bech32/21-brd.md

```diff
@@ -0,0 +1,83 @@
+
+*The following case study contributed by Optech member company [BRD][]
```

Comment:
There are still a lot of instances of 'BRD' in this field report. I think it'd sound more natural if some of them were replaced by we/us.

Reply:
I'm changing this to consistently use pronouns for things their development team did and to use BRD for the software itself.  Hopefully that resolves this concern and @jonatack's concern here: https://github.com/bitcoinops/bitcoinops.github.io/pull/176#discussion_r311051234